### PR TITLE
Gdr 2371

### PR DESCRIPTION
**Pull request by gladkia**
See: https://github.com/gdrplatform/gDRstyle/pull/71
Preview: https://gdrplatform.github.io/gDR/

### CODE REVIEW

**Review by marcin-kam — APPROVED** [review]

File: _pkgdown.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+url: https://gdrplatform.github.io/gDRutils/
```
Comment (j-smola) [inline _pkgdown.yml:1]:
```suggestion
url: https://gdrplatform.github.io/gDR/
```

**Review by j-smola — COMMENTED** [review]
Please, update package version:
gDRstyle 1.1.3
gDRtestData 1.1.7
gDRutils 1.1.5
gDRcore 1.1.11
gDRimport 1.1.4

and
RoxygenNote: 7.3.0

**Comment by gladkia** [comment]
> Please, update package version: gDRstyle 1.1.3 gDRtestData 1.1.7 gDRutils 1.1.5 gDRcore 1.1.11 gDRimport 1.1.4
> 
> and RoxygenNote: 7.3.0

done.

**Comment by j-smola** [comment]
Please, also think how to solve issue with README gDRverse scheme. In pkgdown site it is cropped :( 
_scheme

**Comment by gladkia** [comment]
> Please, also think how to solve issue with README gDRverse scheme. In pkgdown site it is cropped :( _scheme

Fixed.

**Pull request merged**